remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/models/JsonRemoteMessagingConfig.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.remote.messaging.impl.models
+
+data class JsonRemoteMessagingConfig(
+    val version: Long,
+    val messages: List<JsonRemoteMessage>,
+    val rules: List<JsonMatchingRule>
+)
+
+data class JsonRemoteMessage(
+    val id: String,
+    val content: JsonContent?,
+    val exclusionRules: List<Int>?,
+    val matchingRules: List<Int>?
```

Comment:
Why `nullable`. If it is coming from the network, moshi should give you an empty list when these rules are not in the `json`.
I always find confusing when a list can be `null` because it means it may have different semantics of am empty list